### ISSUE TITLE
Corvette - Berserkir 

### ISSUE THREAD

**Issue by ExodistSKY1** [tsunamayo/Skywanderers-Pathfinder-Group-Build#4]
I am not sure if this ship would be considered too curvy

33m length

-Vertical launching missiles near its spine
-Two smaller turrets forward on its left and right 
-Twin Gatlings 

![20200125165023_1](https://user-images.githubusercontent.com/45324484/73128886-59aa4300-3f9c-11ea-8a68-f4ae3a741d07.jpg)

**Comment by tsunamayo** [comment]
Hi, no it looks great! If you could send me the bp so I can have a look at it! Thanks for your contribution.
Cheers

**Comment by ExodistSKY1** [comment]
[z DSF HAAB - Berserkir.zip](https://github.com/tsunamayo/Skywanderers-Pathfinder-Group-Build/files/4126090/z.DSF.HAAB.-.Berserkir.zip)

Here you go!

**Comment by tsunamayo** [comment]
Okay I had a a look at it and it looks great! I have a bunch of remark though:
 - Is it supposed to be dropship of some sort? The front space area look like so. Otherwise front is too low and cramped for a normal living area.
 - Your detailing job looks amazing, but at this stage it could be a waste because of likely modifications
 - Systems wise it has way too many thruster, but I guess you are not done with it!
 - Also I would add a door to the rear portion of the ship, so we can access them, for repair, ect. Maybe put one or two living area, I dont know. Again thinking about the future with crew ect.
 - Need some new door or trap door on the side (using the new block)
 - Need some landing gear also, this thing will look amazing landing properly on a pad.
 - No missiles for now, and when I add missiles back we will have to use a lot less for official ships.

Sorry for the many comments, but I guess this is why I wanted to do a wip section where I can give my feedback early.! You are a very talented builder, your interior work is really amazing, I think once polished this could be the perfect showcase of what is it possible to build in the game!
Cheers, keep it up!

**Comment by ExodistSKY1** [comment]
It is a reworked version from a dropship I had made previously keeping to the same style. That is probably why the interior has that "Dropship" feeling to it. I have not touched the interior yet, I am glad for all your comments because now I know a direction I will go with the interior on this and future builds in this style.

I have not went through and purged the old systems yet, ill try to get to that today.

I plan on using the block missile systems when implemented (less missile number), I can make the area blend well until then.

Update:

-Basic Interior
-Turrets
-updated systems 
-doors added
-missiles removed
-Ramps (too high for just the basic doors would look dumb)
-Landing gear

[z DSF HAAB - Berserkir.zip](https://github.com/tsunamayo/Skywanderers-Pathfinder-Group-Build/files/4132284/z.DSF.HAAB.-.Berserkir.zip)

**Comment by tsunamayo** [comment]
Okay I like the new version, there is actually quite a bit of space inside for such a small volume!

So as the other Berserkir frigate could you get rid of the grill pattern and the pattern over the thruster? It is not a style issue it is more because when viewed at mid distance (which is the way players will view this ship during gameplay) it aliases like crazy!

Also the weapon detailing, while it looks cool, wont be fonctionnal... Also it would be confusion for gameplay. I will give mini-weapon barrel in the next update, so you should be able to use those!
 
You can proceed with paint and interior detailing! What kind of paint pattern had you in mind?
Cheers

**Comment by ExodistSKY1** [comment]
Already updated both ships with the removal of the details you requested! 

I am going to change up the turrets as I am not happy with the way they look yet, and having those barrels will help me decide on the direction I am wanting to go with them! 

I do not have a specific pattern in mind yet, I plan on doing a bunch of different ones and seeing which looks best. I see these ships as from a militaristic style Faction, so they might have a simple and grittier paint scheme to them but we will see what looks best!

**Comment by tsunamayo** [comment]
Yes I can see the militaristic look out of them! But before you settle on some dark-gray mix I would love you to try out something more lively! I think your style could be a perfect fit for some homeworld like vibrant scheme:

![vd2aef32atl31](https://user-images.githubusercontent.com/6604318/73712135-7d177f80-474c-11ea-86ca-b2ca2da08fe3.jpg)

![Homeworld212](https://user-images.githubusercontent.com/6604318/73712145-86a0e780-474c-11ea-8447-a5b77cf85790.jpg)

![C1evjVsXAAEKLbX](https://user-images.githubusercontent.com/6604318/73712152-902a4f80-474c-11ea-8f53-64ff31729b9a.jpg)

![a9bd6e0a7bed7788d315cba662e93808](https://user-images.githubusercontent.com/6604318/73712157-98828a80-474c-11ea-851f-d149075c89b6.jpg)

**Comment by ExodistSKY1** [comment]
Sounds good, I painted a little bit today, I will post up an image a little later as an edit to this post.

The recent update has messed up all the landing gear and anything that was on child entities, you can see this in the ship when trying to bring up the landing gear

**Comment by ExodistSKY1** [comment]
had a try at a paint scheme that looks alright. I have never tried a vibrant color scheme before, it does not look too bad!
![20200205221424_1](https://user-images.githubusercontent.com/45324484/73905782-1e462900-4866-11ea-8261-c378a5751e02.jpg)


Updated:

     -the turrets to look a little smoother 
     -ceilings are all 2.5m with detail
           --holding off on stickers and more of the interior until things are fixed and implemented such                     as beds and other interior things you plan on adding

Broken in the recent update:
     -the hinges such as landing gear and the ramps (worked perfectly before the asteroid update)

[z Corvette V3 - Berserkir.zip](https://github.com/tsunamayo/Skywanderers-Pathfinder-Group-Build/files/4163204/z.Corvette.V3.-.Berserkir.zip)

**Comment by tsunamayo** [comment]
Hi @ExodistSKY1 , I dont know if your ship was impacted by the latest change, and if you have a more recent version to upload. Cheers!

**Comment by ExodistSKY1** [comment]
I think it was impacted pretty heavily, I will be doing a lot of reworking on it due to it blowing up because of heat
